### ISSUE TITLE
bug: stencil does not update the sass styles on Windows 11.

### ISSUE THREAD

**Issue by a7medm7med** [ionic-team/stencil#4130]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/ionic-team/stencil/blob/main/.github/CONTRIBUTING.md).
- [X] I agree to follow the [Code of Conduct](https://github.com/ionic-team/stencil/blob/main/CODE_OF_CONDUCT.md).
- [X] I have searched for [existing issues](https://github.com/ionic-team/stencil/issues) that already report this problem, without success.

### Stencil Version

3.0.0

### Current Behavior

When starting using the `sass` in a stencil project and trying to import other styles from modules using the `@use()` rule on making any changes in the files no thing updates.

Try cloning my repo and changing the `background-color` in the `my-component.scss` file to see that nothing will be updated until rebuild the project.

IMPORTANT: This bug is only in Windows, not macOS or Linux.

### Expected Behavior

Stencil looks for changes that happen in the imported sass modules during the development process.

### System Info

```shell
System: node 18.14.2
     Plaform: windows (10.0.22624)
   CPU Model: 11th Gen Intel(R) Core(TM) i5-11400H @ 2.70GHz (12 cpus)
    Compiler: D:\stencil-component-starter-main\node_modules\@stencil\core\compiler\stencil.js
       Build: 20230228172721
     Stencil: 3.1.0
  TypeScript: 4.9.5
      Rollup: 2.42.3
      Parse5: 7.1.2
      Sizzle: 2.42.3
      Terser: 5.16.4
```

### Steps to Reproduce

1. Set up a stencil component project.
2. Use the sass modules by using the `@use()` rule.
3. Start the project.
4. Try making any changes in the imported file and changes will not apply until rebuild the project,

### Code Reproduction URL

https://github.com/a7medm7med/stencil-component-starter

**Comment by rwaskiewicz** [comment]
Thanks @a7medm7med!

I was able to confirm this isn't working in Windows (in non-WSL environments, for future readers). I'm going to label this accordingly. Thanks again!

**Comment by christian-bromann** [comment]
@a7medm7med a fix for this has been released as a part of [Stencil v4.11.0 release](https://github.com/ionic-team/stencil/releases/tag/v4.11.0).
